test/regression/aarch64/instructions/neon.cc

```diff
@@ -368,6 +368,50 @@ TEST_P(InstNeon, and) {
 }
```

Comment:
I working under the assumption that each new instruction has an associated set of tests

Reply:
Yes they should do, apart from the BRK instruction for reasons I mentioned in the other comment